.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ name: CI with Composer scripts
 on:
   push:
     branches:
-      - main
+      - main  
```

Comment:
it's a typo or it's intended?

Reply:
Maybe one of the linters adds that automatically?